docs/reference/transform/apis/delete-transform.asciidoc

```diff
@@ -19,10 +19,15 @@ Deletes an existing {transform}.
 ==== {api-prereq-title}
 
 * Before you can delete the {transform}, you must stop it.
-* If the {es} {security-features} are enabled, you must have `manage_transform` 
-cluster privileges to use this API. The built-in `transform_admin` role has 
-these privileges. For more information, see <<security-privileges>> and 
-<<built-in-roles>>.
+
+If the {es} {security-features} are enabled, you must have the following 
```

Comment:
Strictly speaking, you only need the `manage_transform` cluster privilege, which is included in the built-in `transform_admin` role. Maybe it would better to only include  `manage_transform` in the bulleted list, and then follow with the sentence 'The built-in `transform_admin` role has this privilege'. as before? Same applies to the other pages below.